src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -82,6 +82,7 @@ def create_subnet(resource_group_name, virtual_network_name, subnet_name,
     subnet.address_prefix = address_prefix
 
     if network_security_group:
+        from azure.mgmt.network.models import NetworkSecurityGroup
```

Comment:
>rom azure.mgmt.network.models import NetworkSecurityGroup [](start = 9, length = 57)

it might be more consistent with the rest of the code base to have this at the top, unless it is especially expensive for some reason